install/azure.go

```diff
@@ -51,12 +56,20 @@ type aksClusterInfo struct {
 	NodeResourceGroup string `json:"nodeResourceGroup"`
 }
 
-type accountInfo struct {
-	ID   string `json:"id"`
-	Name string `json:"name"`
+func (k *K8sInstaller) aksSetup(ctx context.Context) error {
+	if err := k.azureRetrieveSubscriptionID(ctx); err != nil {
+		return err
+	}
+
+	if err := k.azureRetrieveAksNodeResourceGroup(ctx); err != nil {
+		return err
+	}
+
+	err := k.azureSetupServicePrincipal(ctx)
```

Comment:
```suggestion
	return k.azureSetupServicePrincipal(ctx)
```

Reply:
Thanks, missed that during the refactor. Fixed in latest push.